aws_quickstart/datadog_integration_api_call_v2.yaml

```diff
@@ -66,160 +66,194 @@ Resources:
       ManagedPolicyArns:
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
   # Retrieving secrets passed in via SecretsManager Arn
-  DatadogAPICall:
-    Type: "Custom::DatadogAPICall"
+  DatadogAPIHandler:
```

Comment:
I think we should name this something that sounds more like a resource or connection.  This will appear in the stack and it represents the link that integrates the encompassing AWS account to Datadog.  It's not particularly related to the API, other than the fact that the handler/serviceToken interact with the API.